app/views/reports/admin_dashboard.html.erb

```diff
@@ -0,0 +1,51 @@
+<html lang="<%= @locale %>">
+<% title t('admin.dashboard') %>
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
Is Bootstrap necessary?

Reply:
This is with Bootstrap. I have also added hover state.
![with bootstrap](https://user-images.githubusercontent.com/32712346/45315462-1e97a300-b552-11e8-921e-629af2a4e859.png)

This is without Bootstrap.
![without bootstrap](https://user-images.githubusercontent.com/32712346/45315467-20616680-b552-11e8-8ebe-95e34fbccd27.png)

Which on do you think would be better?